doc/py_tutorials/py_feature2d/py_features_harris/py_features_harris.markdown

```diff
@@ -40,12 +40,12 @@ using **cv.Sobel()**).
 Then comes the main part. After this, they created a score, basically an equation, which
 determines if a window can contain a corner or not.
 
-\f[R = det(M) - k(trace(M))^2\f]
+\f[R = \det(M) - k(\operatorname{trace}(M))^2\f]
```

Comment:
BTW, proposed changes can be viewed here: http://pullrequest.opencv.org/buildbot/export/pr/20158/docs/dc/d0d/tutorial_py_features_harris.html

Reply:
Awesome! Thanks for the information!